run/jobs/e2e_test.py

```diff
@@ -36,36 +36,61 @@
 @pytest.fixture
 def setup_job():
     # Build container image and run the job
-    subprocess.check_call(
-        [
-            "gcloud",
-            "builds",
-            "submit",
-            "--config",
-            "e2e_test_setup.yaml",
-            "--project",
-            PROJECT,
-            "--substitutions",
-            "_SERVICE=" + SERVICE + ",_VERSION=" + SUFFIX + ",_REGION=" + REGION,
-        ]
-    )
+    # Retry up to 3 times
+    built = False
+    for x in range(3):
```

Comment:
Like David suggested, would it be possible to use [`backoff`](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#retry-rpcs)?

Reply:
Done.